mp4/saiz_test.go

```diff
@@ -5,6 +5,6 @@ import (
 )
 
 func TestSaiz(t *testing.T) {
-	saiz := &SaizBox{}
+	saiz := &SaizBox{DefaultSampleInfoSize: 1}
```

Comment:
A bit strange test with zero sampleCount, but that is nothing new. I can fix that in another PR.

Reply:
For some reason the simple test started breaking with my other fixes, but it appeared an empty box didn't really make sense anyways.